sentiment_analyzer.py

```diff
@@ -44,7 +44,12 @@ def compute_sentiment(self, country, city, hashtag):
             for tweet in tweets:
                 compound_score = self.sentiment_analyzer(tweet)
                 compound_sum += compound_score
-            return compound_sum / num_tweets
+            avg_score = compound_sum / num_tweets
+            if avg_score <= 0.5 and avg_score >= -0.5:
```

Comment:
Change this to 0.25 instead of 0.5, and -0.25 for -0.5

Reply:
Updated and added the weight to config